addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -70,7 +70,7 @@ def _model_sorting_key(self, leave_type):
     virtual_leaves_taken = fields.Float(
         compute='_compute_leaves', string='Virtual Time Off Already Taken',
         help='Sum of validated and non validated time off requests.')
-    closest_allocation_to_expire = fields.Many2one('hr.leave.allocation', 'Allocation', compute='_compute_leaves')
```

Comment:
Changing the name here requires the `.po` file to be updated :innocent: 
I guess we could manage to not change it, what do you think? :-D